etc/config/c++.amazon.properties

```diff
@@ -1223,6 +1223,12 @@ libs.expected_lite.versions.010.version=0.1.0
 libs.expected_lite.versions.010.path=/opt/compiler-explorer/libs/expected-lite/v0.1.0/include
 libs.expected_lite.versions.trunk.version=trunk
 libs.expected_lite.versions.trunk.path=/opt/compiler-explorer/libs/expected-lite/trunk/include
+libs.highway.name=Highway
+libs.highway.versions=trunk
+libs.highway.url=https://github.com/google/highway
+libs.highway.versions.trunk.version=trunk
```

Comment:
So, in the `infra` PR you sent we are only pulling down the version you specified there: `0.12.2`. That means the path is `/opt/compiler-explorer/libs/highway/0.12.2`.  To have a trunk version too we'll need to update the infra pull, and to support 0.12.2 here you'll need to list it separately.

What version(s) would you like?

Reply:
Ah, right. Would be nice to have both, I've sent another PR for infra, does this work?

Reply:
Absolutely: thanks (and sorry it wasn't obvious)